ttnn/cpp/ttnn/operations/moreh/moreh_nll_loss_backward/moreh_nll_loss_backward_pybind.cpp

```diff
@@ -0,0 +1,38 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "moreh_nll_loss_backward_pybind.hpp"
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "moreh_nll_loss_backward.hpp"
+#include "pybind11/cast.h"
+#include "ttnn/cpp/pybind11/decorators.hpp"
+
+namespace py = pybind11;
+
+namespace ttnn::operations::moreh::moreh_nll_loss_backward {
+
+void bind_moreh_nll_loss_backward_operation(py::module &module) {
+    bind_registered_operation(
+        module,
+        ttnn::moreh_nll_loss_backward,
+        R"doc(moreh_nll_loss_backward(target_tensor: ttnn.Tensor, weight_tensor: Optional[ttnn.Tensor], output_grad_tensor: ttnn.Tensor, input_grad_tensor: Optional[ttnn.Tensor], ignore_index: int32, memory_config: Optional[ttnn.MemoryConfig] = None, compute_kernel_config: Optional[DeviceComputeKernelConfig]) -> ttnn.Tensor
```

Comment:
```suggestion
        R"doc(moreh_nll_loss_backward(target_tensor: ttnn.Tensor, weight_tensor: Optional[ttnn.Tensor], output_grad_tensor: ttnn.Tensor, input_grad_tensor: Optional[ttnn.Tensor], ignore_index: int, memory_config: Optional[ttnn.MemoryConfig] = None, compute_kernel_config: Optional[DeviceComputeKernelConfig]) -> ttnn.Tensor
```
I think the type mentioned in the document refers to the Python type. Since int in Python may not always be int32, it might be better to specify it as int instead of int32

Reply:
Updated